Symfony/CS/Fixer/Contrib/ShortBoolCastFixer.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Fixer\Contrib;
+
+use Symfony\CS\AbstractFixer;
+use Symfony\CS\Tokenizer\Token;
+use Symfony\CS\Tokenizer\Tokens;
+
+class ShortBoolCastFixer extends AbstractFixer
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function fix(\SplFileInfo $file, $content)
+    {
+        $tokens = Tokens::fromCode($content);
+        for ($i = count($tokens) - 1; $i > 1; --$i) {
+            if ($tokens[$i]->equals('!')) {
```

Comment:
it would be cleaner to return early (continue in this case) here instead of going to nest 5+ levels deep.